Azure/sonic-buildimage
# Add the version files for the branch 202012

#### Why I did it
Add the version file for reproducible build for 202012

#### How I did it

#### How to verify it
make SONIC_VERSION_CONTROL_COMPONENTS=all <targets>
make freeze
Expect not version change after the version files added.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
